content/shmem_collect.tex

```diff
@@ -110,17 +110,6 @@
     \end{itemize}
 }
 
-\apidesctable{
```

Comment:
@jdinan just to make sure i understand -- this isn't a semantic change because this table was already deleted, and the teams merge accidentally added it back in? is that correct?

Reply:
No, this was overlooked during the great Fortran purge (which is why it should be fair game for section committee).  This should not result in a semantic change.  If you prefer, I can handle this as part of #276.

Reply:
Moved this edit over to #276 via fe6e1e3